src/awx_plugins/credentials/injectors.py

```diff
@@ -16,7 +19,7 @@
 import yaml
 
 
-def aws(cred, env, private_data_dir):
+def aws(cred: Credential, env: dict, private_data_dir: str):
```

Comment:
While on it, could you also mark the returned type for all of the functions in the module?
```suggestion
def aws(cred: Credential, env: dict, private_data_dir: str) -> None:
```